.azure/build-win.yml

```diff
@@ -22,7 +22,7 @@ parameters:
     default: true
   - name: build_shared
     type: boolean
-    default: false
+    default: true
```

Comment:
I would keep this on default `false` to be consistent with the non-Windows template. Otherwise people will get confused in the future.

Reply:
Yeah, sure, no problem. I had the default `vcpkg_target_triplet` set to `x64-windows`, and I wanted to make those two consistent. I've now changed the default `vcpkg_target_triplet` to `x64-windows-static-md` to make it consistent in that way.